<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_Links">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Backward">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentDistabce">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Obstacle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_resetSensor">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Forward">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>41</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>For</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Rechts">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdN_CurrentDistance">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_activateSensor">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Activ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_deactivateSensor">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Deac</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Stop">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(239, 41, 41);
gridline-color: rgb(239, 41, 41);
border-color: rgb(239, 41, 41);</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Sure">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Sure</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>1</number>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(239, 41, 41);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DSB_setSpeed">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>72</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <double>50.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="setSpeed">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>setSpeed</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DSB_KPL">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>50.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.001000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DSB_KIL">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>50.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DSB_KDL">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>50.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DSB_KDR">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>50.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DSB_KIR">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>50.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DSB_KPR">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>50.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="KPL">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>KPL</string>
    </property>
   </widget>
   <widget class="QLabel" name="KDL">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>KDL</string>
    </property>
   </widget>
   <widget class="QLabel" name="KIL">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>KIL</string>
    </property>
   </widget>
   <widget class="QLabel" name="KPR">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>KPR</string>
    </property>
   </widget>
   <widget class="QLabel" name="KIR">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>KIR</string>
    </property>
   </widget>
   <widget class="QLabel" name="KDR">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>KDR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Start">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Cancel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DSB_setDistance">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>72</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <double>500.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="setDistance">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>setDistance</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textOrderID">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="OrderID">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OrderID</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textOrderDescription">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="OrderDescription">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textProzess">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Piboto Light'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Roboter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Roboter</string>
    </property>
   </widget>
   <widget class="QLabel" name="fahrtzu">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>fährt zu</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textOrderListDescription">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="OrderListID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OrderListID</string>
    </property>
   </widget>
   <widget class="QLabel" name="OrderListDescription">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textOrderListID">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_21">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_22">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_23">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_24">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Try">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Try</string>
    </property>
   </widget>
   <widget class="QLabel" name="PathXML">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PathXML</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPathXML">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>301</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_OKXML">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_WakeUp">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>340</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>wake up</string>
    </property>
   </widget>
   <widget class="Line" name="line_25">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_27">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>290</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_26">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_28">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_29">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
